k8s/helmfile/env/staging/api.values.yaml.gotmpl

```diff
@@ -1,5 +1,5 @@
 image:
-  tag: 8x.33.0
+  tag: 8x.34.0
 
```

Comment:
Is the staging conf missing the `storage` section?

Reply:
Thing is I don't really know the values staging and production want yet, so I was planning to add this while dibsing staging. Sorry for not mentioning it yet.

Reply:
ah that sounds reasonable, hard to know how GCP behaves otherwise :+1: